### DISCUSSION TITLE
allowHover equivalent for Dropdowns in NextUI?

### DISCUSSION THREAD

**Discussion by goustaff** [heroui-inc/heroui#1806] [Feature requests]
Hello!
I migrated from _Material Tailwind_ to _NextUI_.

I appreciate the fact that the integration is closer to the core of my application, and performs better in terms of pagespeed.

However, there is one point that disappoints me a little concerning a feature: the ability to reproduce the `allowHover` behaviour of _Material Tailwind_ doesn't seem to be native with _NextUI_.

The desired behaviour is very simple to explain:
- When the mouse hovers over the menu, it opens
- When the mouse stops hovering the menu, it closes.

How do you reproduce this with NextUI?

Thank you in advance for your answers.

---

Material Tailwind related documentation and example:
- https://www.material-tailwind.com/docs/react/menu#menu-custom-list

NextUI related documentation and (no) example:
- https://nextui.org/docs/components/dropdown
- https://nextui.org/docs/components/popover

**Comment by goustaff** [comment]
Oops, similar to:
https://github.com/nextui-org/nextui/discussions/1806

**Comment by timomedia** [comment]
I also hope to add this option to the dropdown
